uma/protocol/settlement.py

```diff
@@ -0,0 +1,41 @@
+# Copyright ©, 2022-present, Lightspark Group, Inc. - All Rights Reserved
+
+from dataclasses import dataclass
+
+from uma.JSONable import JSONable
+
+
+@dataclass
+class SettlementAsset(JSONable):
+    identifier: str
+    """
+    The identifier of the asset. For Spark, this is the token public key.
+    For Lightning/BTC, this should be "BTC".
```

Comment:
Are we expecting that people have to include lightning here? I'd been thinking that lightning support was always required by default and the other options would just be additional.